wzdx/standard_to_enhanced/planned_events_translator.py

```diff
@@ -4,10 +4,11 @@
 import logging
```

Comment:
Light on unit tests for this file

Reply:
now at 88%, mostly missing individual lines and the main method